syft/sandbox.py

```diff
@@ -1,4 +1,6 @@
 import importlib
+from tensorflow.keras import datasets
```

Comment:
tensorflow is not a direct dependency of pysyft, could you make it optional? You can use what is done in `syft/frameworks/__init__.py`

Reply:
Following our discussion over the slack, I did the required changes